llvm/test/CodeGen/X86/vector-compare-all_of.ll

```diff
@@ -1543,34 +1543,49 @@ define i1 @select_v2i8(ptr %s0, ptr %s1) {
 ; SSE2-NEXT:    punpcklbw {{.*#+}} xmm1 = xmm1[0,0,1,1,2,2,3,3,4,4,5,5,6,6,7,7]
 ; SSE2-NEXT:    pshuflw {{.*#+}} xmm0 = xmm1[0,0,1,1,4,5,6,7]
 ; SSE2-NEXT:    pshufd {{.*#+}} xmm0 = xmm0[0,0,1,1]
-; SSE2-NEXT:    movmskpd %xmm0, %eax
-; SSE2-NEXT:    cmpl $3, %eax
-; SSE2-NEXT:    sete %al
+; SSE2-NEXT:    movmskpd %xmm0, %ecx
+; SSE2-NEXT:    movl %ecx, %eax
+; SSE2-NEXT:    andb $2, %al
+; SSE2-NEXT:    shrb %al
+; SSE2-NEXT:    andb %cl, %al
```

Comment:
I'm not certain but I think these regressions (and vector-compare_any_of.ll) will go away with #87480

Reply:
@bjope  #87480 has landed if you want to rebase and see if we have any improvements